README.md

```diff
@@ -63,6 +63,20 @@ documents all of these.  The most commonly used flags:
 * `-Dmodernizer.failOnViolations` - fail phase if violations detected, defaults to true
 * `-Dmodernizer.skip` - skip plugin execution, defaults to false
 
+Suppression
+-----------
+To suppress modernizer errors, add the following to the `<dependencies>` stanza in your pom.xml
```

Comment:
I think that this section should start with info about the `@SuppressWarnings("modernizer")` stuff: that's what's actually useful to people in most cases. Adding the dependency just enables that.